src/app/pages/task-calendar/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -1175,7 +1175,12 @@ export class ReplicationWizardComponent {
             this.ws.call('replication.count_eligible_manual_snapshots', payload).subscribe(
                 (res) => {
                     this.eligibleSnapshots = res.eligible;
-                    this.snapshotsCountField.paraText = '<span class="' + (res.eligible == 0 ? 'warnning-paragraph' : 'info-paragraph' )+'"><b>' + res.eligible + '</b> snapshots found</span>';
+                    const isPush = this.entityWizard.formArray.controls[0].controls['source_datasets_from'].value === 'local';
+                    this.snapshotsCountField.paraText = '<span class="' +
+                    (res.eligible == 0 ? (isPush ? 'info-paragraph' : 'warnning-paragraph') : 'info-paragraph' ) +
```

Comment:
Why is this consistently misspelled: "warnning-paragraph"?
The assignment here is very long and it makes the logic hard to read. Assigning values before building the string would be clearer.  Also assigning defaults first and using string interpolation:
(untested)
```
spanclass = 'info-paragraph';
snapexpl = '';
if  (res.eligible === 0 && isPush) {
    spanclass = 'warning-para';
    snapexpl = 'Snapshots will be created automatically.';
}
this.snapshotsCountFieldparaText = `<span class="#{spanclass}"><b>#{res.eligible}</b> snapshots found. #{snapexpl}</span>`;
```

Reply:
thanks @wblock, pushed a commit to fix it :)